langgraph_supervisor/agent_name.py

```diff
@@ -62,13 +63,13 @@ def remove_inline_agent_name(message: BaseMessage) -> BaseMessage:
     if not isinstance(message, AIMessage) or not message.content:
         return message
 
-    is_content_blocks_content = _is_content_blocks_content(message.content)
-    if is_content_blocks_content:
-        text_blocks = [block for block in message.content if block["type"] == "text"]
+    if is_content_blocks_content := _is_content_blocks_content(message.content):
+        content_blocks_content = message.content
```

Comment:
why change this?

Reply:
I thought it was necessary to get around a mypy error, but it looks like it's not. Removed.